ci/resources/phpfpm/php-fpm.conf

```diff
@@ -0,0 +1,537 @@
+;;;;;;;;;;;;;;;;;;;;;
```

Comment:
What about putting the interesting lines? All these comments aren't useful (we could just keep a link to the source) and they make it harder to read and look for the interesting stuff.